src/clue/components/clue-app-header.tsx

```diff
@@ -49,7 +49,7 @@ export class ClueAppHeaderComponent extends BaseComponent<IProps> {
           </div>
           <div className="separator"/>
           <CustomSelect
-            items={[{text: `${problem.title}: ${problem.subtitle}`}]}
+            items={[{text: `${problem.title}${problem.subtitle ? `: ${problem.subtitle}`: ""}`}]}
```

Comment:
No, because the point is to add the `: ` only when there's a non-empty subtitle. I changed it when I encountered a case in testing which didn't have a subtitle and `Title: ` with no subtitle looks stupid. 😉